Description: Changes from webupd8.
Author: Alin Andrei <webupd8@gmail.com>
Last-Revision: 2013-09-11
--- a/MediterraneanTribute/gtk-2.0/gtkrc
+++ b/MediterraneanTribute/gtk-2.0/gtkrc
@@ -530,7 +530,7 @@
 	engine "murrine" {
         gradient_shades		= {1.01, 1.00, 1.00, 1.00}
 		border_shades 		= {1.6, 1.6}
-        roundness           = 2
+        roundness           = 3
         contrast            = 0.0           # Contraste separadores
 		textstyle           = 1
 		text_shade			= 1.1
--- a/MediterraneanTribute/gtk-3.0/gtk.css
+++ b/MediterraneanTribute/gtk-3.0/gtk.css
@@ -5,7 +5,7 @@
 @define-color fg_color #363636;
 @define-color selected_bg_color #507fc5;
 @define-color selected_fg_color #ffffff;
-@define-color tooltip_bg_color #343434;
+@define-color tooltip_bg_color #040404;
 @define-color tooltip_fg_color #ffffff;
 
 /* Colormap actually used by the theme, to be overridden in other css files */
--- a/MediterraneanTributeBlue/gtk-2.0/gtkrc
+++ b/MediterraneanTributeBlue/gtk-2.0/gtkrc
@@ -464,9 +464,9 @@
 	ythickness	= 0
 
     fg[NORMAL]          = @theme_fg_dark_color
-	fg[PRELIGHT]	    = @theme_fg_dark_color
-	fg[ACTIVE]	        = @theme_fg_dark_color
-	fg[SELECTED]	    = @theme_fg_dark_color
+	fg[PRELIGHT]	    = @selected_fg_color
+	fg[ACTIVE]	        = @selected_fg_color
+	fg[SELECTED]	    = @selected_fg_color
 
     bg[NORMAL]          = shade (1.00, @theme_bg_dark_color)
     
@@ -515,25 +515,23 @@
 	}
 }
 
-style "menubar_menuitem" {
+style "menu_item_dark" {
 	xthickness	= 2
 	ythickness	= 3
 
-    fg[NORMAL]          = @fg_color
-	fg[PRELIGHT]	    = @fg_color
-	fg[ACTIVE]	        = @fg_color
-	fg[SELECTED]	    = @fg_color
+    fg[NORMAL]          = @theme_fg_dark_color
 
-    bg[SELECTED]      	= shade(1.03, @bg_color)
-    bg[PRELIGHT]      	= shade(1.03, @bg_color)
+	fg[PRELIGHT]	    = @selected_fg_color
+	fg[ACTIVE]	        = @selected_fg_color
+	fg[SELECTED]	    = @selected_fg_color
 
 	engine "murrine" {
-        gradient_shades		= {1.01, 1.00, 1.00, 1.00}
-		border_shades 		= {1.6, 1.6}
+        gradient_shades		= {0.90, 0.95, 0.95, 1.00}
+		border_shades 		= {1.40, 1.0}
         roundness           = 2
-        contrast            = 0.0           # Contraste separadores
+        contrast            = 0.8           # Contraste separadores
 		textstyle           = 1
-		text_shade			= 1.1
+		text_shade			= 1.15
 	}
 }
 
@@ -872,8 +870,7 @@
 widget_class "*<GtkMenu>*"			    style "menu"
 widget_class "*<GtkMenuItem>*"			style "menu_item_white"
 #widget_class "*<GtkMenuBar>.<GtkMenuItem>*"	style "menu_item_white"
-widget_class "*<GtkMenuBar>.<GtkMenuItem>*"	style "menubar_menuitem"
-widget_class "*<GtkMenuBar>*<GtkMenuItem>*" style "menubar_menuitem"
+widget_class "*<GtkMenuBar>.<GtkMenuItem>*"	style "menu_item_dark"
 
 widget_class "*<GtkToolbar>"                    style "toolbar"
 widget_class "*<GtkHandleBox>"                  style "toolbar"
--- a/MediterraneanTributeBlue/gtk-3.0/gtk.css
+++ b/MediterraneanTributeBlue/gtk-3.0/gtk.css
@@ -5,7 +5,7 @@
 @define-color fg_color #363636;
 @define-color selected_bg_color #2e91be;
 @define-color selected_fg_color #ffffff;
-@define-color tooltip_bg_color #343434;
+@define-color tooltip_bg_color #040404;
 @define-color tooltip_fg_color #ffffff;
 
 /* Colormap actually used by the theme, to be overridden in other css files */
@@ -14,7 +14,7 @@
 @define-color theme_text_color @text_color;
 @define-color theme_fg_color @fg_color;
 @define-color theme_shadow_color alpha(#fff, 0.22);
-@define-color theme_selected_bg_color #2e91be;
+@define-color theme_selected_bg_color shade(#2e91be, 1.00);
 @define-color theme_selected_fg_color @selected_fg_color;
 @define-color theme_selected_shadow_color alpha(#000, 0.12);
 @define-color theme_tooltip_bg_color @tooltip_bg_color;
